app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+  }
+}) }}
+
+## Volunteering with children and young people
+
+Candidates can update their answers before (and 5 working days after) submitting their application.
+
+{{ appProtocol({
+  title: "Candidate has school experience",
+  items: {
+    question: "Do you have experience volunteering with young people or in school?",
+    required: "Yes",
+    why: [{
+      who: ["service"],
+      what: "needs to offer guidance about completing this section and enable branching; if no experience of volunteering with young people or in school, we direct candidates to the [Get school experience](https://schoolexperience.education.gov.uk) service"
+    }],
+    what: ["Answer not submitted as part of application"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Role - Title",
+  items: {
+    question: "Your role",
+    required: "No",
+    api: {
+      param: "work_experience.volunteering[].role",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
+    },
+    why: [{
+      who: ["service"],
+      what: "uses this label to distinguish between different roles"
+    }, {
+      who: ["provider"],
+      what: "uses this to build a picture of a candidate’s experience"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Reference from an employer may collaborate some degree of information"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Role - Organisation",
+  items: {
+    question: "Organisation where you gained experience or volunteered",
+    required: "No",
+    api: {
+      param: "work_experience.volunteering[].organisation_name",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
+    },
+    why: [{
+      who: ["service"],
+      what: "uses this label to distinguish between different roles"
+    }, {
+      who: ["provider"],
+      what: "uses this to build a picture of a candidate’s background"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Reference may collaborate some of this information"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Role - Dates of volunteering",
+  items: {
+    question: "Start date
+      End date",
+    required: "No",
+    api: {
+      param: "work_experience.volunteering[].start_date
+      work_experience.volunteering[].end_date",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "uses this to build a picture of a candidate’s background"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Reference may collaborate some of this information"]
```

Comment:
Again, we check the range is a valid one, and we don't care about day of the month.